platform/accessibility/guidance/staging-review-processes.md

```diff
@@ -252,6 +268,94 @@ Unresolved content resize issues are documented in issue tickets and listed as k
 *   [F80: Failure of Success Criterion 1.4.4 when text-based form controls do not resize when visually rendered text is resized up to 200%](http://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/F80)
 
 
+### Pointer Gestures
+
+**Who:** Frontend engineer
+
+**When:** As part of daily development
+
+#### Steps to test
+
+*   Use a mobile device to test a functionality of feature
+    *   Make sure you can fully utilize feature using only 1 finger to gesture (tapping, swiping, etc)
```

Comment:
Combine the main and sub-bullet to change to: Using a mobile device, make sure you can use the full functionality of a feature with only 1 finger to gesture (tapping, swiping, etc.)